paimon-core/src/test/java/org/apache/paimon/operation/KeyValueFileStoreScanTest.java

```diff
@@ -148,20 +148,20 @@ public void testWithKeyFilter() throws Exception {
     }
 
     @Test
-    public void testWithValueFilter() throws Exception {
+    public void testFilterBucket() throws Exception {
```

Comment:
The test uses `withValueFilter`, but its name is testFilterBucket, which is strange.

Reply:
-> testWithValueFilterBucket